blocks/commerce-customer-information/commerce-customer-information.css

```diff
@@ -0,0 +1 @@
+/* stylelint-disable no-empty-source */
```

Comment:
Are these empty CSS files necessary?

Reply:
I would likely say yes, they are necessary. The CSS for blocks is imported automatically, so if we remove those empty files, we would constantly see 404 errors in the console, which could be confusing.

<img width="1062" alt="Screenshot 2024-11-04 at 14 46 31" src="https://github.com/user-attachments/assets/e857d80f-5e74-4f2b-b5ae-6186f317e0b9">